### PR TITLE
Update Belgium holidays: add bank holidays

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Update Belgium holidays: add bank holidays.
It's a bit strange, but Dec 26 holiday name in German, Dutch and French is "Bank Holiday" ([DE](https://www.nbb.be/de/die-nationalbank/die-nationalbank/arbeitsfreie-tage), [NL](https://www.nbb.be/nl/over-de-nationale-bank/de-nationale-bank/sluitingsdagen), [FR](https://www.nbb.be/fr/la-banque-nationale/la-banque-nationale/jours-de-fermeture)).

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/belgium.py [hunk 1]
```diff
@@ -70,6 +72,16 @@ def _populate(self, year):
...
 
+    def _populate_bank_holidays(self):
+        # Good Friday.
+        self._add_good_friday(tr("Goede Vrijdag"))
+
+        # Friday after Ascension Day.
+        self._add_holiday(tr("Vrijdag na O. L. H. Hemelvaart"), self._easter_sunday + td(days=+40))
+
+        # Bank Holiday.
+        self._add_christmas_day_two(tr("Bank Holiday"))
```
Comment (arkid15r) [inline holidays/countries/belgium.py:83]:
```suggestion
        self._add_christmas_day_two(tr("Banksluitingsdag"))
```

File: holidays/locale/en_US/LC_MESSAGES/BE.po [hunk 2]
```diff
@@ -63,3 +63,15 @@ msgstr "Armistice Day"
...
+#. Good Friday.
+msgid "Goede Vrijdag"
+msgstr "Good Friday"
+
+#. Friday after Ascension Day.
+msgid "Vrijdag na O. L. H. Hemelvaart"
+msgstr "Friday after Ascension Day"
+
+#. Bank Holiday.
+msgid "Bank Holiday"
```
Comment (arkid15r) [inline holidays/locale/en_US/LC_MESSAGES/BE.po:76]:
```suggestion
msgid "Banksluitingsdag"
```

File: holidays/locale/fr/LC_MESSAGES/BE.po [hunk 3]
```diff
@@ -63,3 +63,15 @@ msgstr "Jour de l'Armistice"
...
+msgid "Goede Vrijdag"
+msgstr "Vendredi Saint"
+
+#. Friday after Ascension Day.
+msgid "Vrijdag na O. L. H. Hemelvaart"
+msgstr "Vendredi suivant l'Ascension"
+
+#. Bank Holiday.
+msgid "Bank Holiday"
+msgstr "Bank Holiday"
```
Comment (arkid15r) [inline holidays/locale/fr/LC_MESSAGES/BE.po:77]:
```suggestion
msgid "Banksluitingsdag"
msgstr "Jour de fermeture bancaire"
```

File: holidays/locale/de/LC_MESSAGES/BE.po [hunk 4]
```diff
@@ -63,3 +63,15 @@ msgstr "Waffenstillstand"
...
+msgid "Goede Vrijdag"
+msgstr "Karfreitag"
+
+#. Friday after Ascension Day.
+msgid "Vrijdag na O. L. H. Hemelvaart"
+msgstr "Freitag nach Christi Himmelfahrt"
+
+#. Bank Holiday.
+msgid "Bank Holiday"
+msgstr "Bank Holiday"
```
Comment (arkid15r) [inline holidays/locale/de/LC_MESSAGES/BE.po:77]:
```suggestion
msgid "Banksluitingsdag"
msgstr "Bankschlusstag"
```

**Review by arkid15r — COMMENTED** [review]
I found a possibly better translation for the "Bank Holiday" [here](https://www.kbcbrussels.be/retail/en/information/bank-holidays.html) and then translated it from NL for DE. Might be it 🤷 

🇧🇪 LGTM

File: holidays/countries/belgium.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/belgium.py]: Done.

**Review by arkid15r — APPROVED** [review]
🚢

**Pull request merged**